src/ai/backend/manager/api/service.py

```diff
@@ -163,7 +163,7 @@ async def get_info(request: web.Request) -> web.Response:
 @check_api_params(
     t.Dict(
         {
-            tx.AliasedKey(["name", "clientSessionToken"])
+            tx.AliasedKey(["name", "session_name", "clientSessionToken"])
```

Comment:
This should be `service_name`.
```suggestion
            tx.AliasedKey(["name", "service_name", "clientSessionToken"])
```

Reply:
Thanks to reviewing my mistake.